packages/core/src/actionTypes.ts

```diff
@@ -18,3 +18,4 @@ export const error = ActionTypes.ErrorCustom;
 export const update = ActionTypes.Update;
 export const choose = ActionTypes.Choose;
 export const pure = ActionTypes.Pure;
+export const forEach = ActionTypes.ForEach;
```

Comment:
if the actions creator is named `each` it would probably be best to reflect that here as well so we use the unified terminology in the codebase, docs, etc